index.js

```diff
@@ -141,7 +141,8 @@ app.on('ready', () => {
     });
 
     rpc.on('exit', ({ uid }) => {
-      sessions.get(uid).exit();
+      const session = sessions.get(uid);
+      session ? session.exit() : false;
```

Comment:
Why not `session && session.exit()` if you want to be less verbose (since you’re not using `if`)?